appveyor.yml

```diff
@@ -98,6 +98,49 @@ environment:
       HDF5_DIR: "C:\\hdf5\\%HDF5_VERSION%\\%HDF5_VSVERSION%"
       HDF5_VERSION: "1.10.1"
 
```

Comment:
This appears to be duplicated?

Reply:
Good catch, probably I made a mistake with the rebase.